airflow/config_templates/default_airflow.cfg

```diff
@@ -91,6 +91,10 @@ default_timezone = utc
 # SequentialExecutor, LocalExecutor, CeleryExecutor, DaskExecutor, KubernetesExecutor
 executor = SequentialExecutor
 
+# Used only with InProcessExecutor. If set to True DAG will fail with first
+# failed task. Helpful for debugging purposes.
+fail_fast = False
```

Comment:
I think for now this should be a conf in "InProcessExecutor" section for now rather than generic conf. Later when/if we introduce fail fast globally we can make it global conf.

Reply:
How about [debug] sections simpler and can be reused?

Reply:
Yep. Why not. We can add more stuff there then in this section.